lib/crypto/tasks.py

```diff
@@ -264,6 +268,11 @@ def file_supports_firefox(version):
         to_unsign = version.files.filter(ff_version_filter,
```

Comment:
ahhhh, yes very good catch, I see it now, it should indeed be using `to_unsign`, will fix this straight away, thanks!